CIPs/CIP-23/CIP-23.md

```diff
@@ -0,0 +1,176 @@
+  "type": "array",
+  "title": "AlsoKnownAs",
+  "items": {
+    "$ref": "#/definitions/Account"
+  },
+  "definitions": {
+    "Attestation": {
+      "type": "object",
+      "properties": {
+        "did-jwt": {
+          "type": "string",
+          "maxLength": 1000
+        },
+        "did-jwt-vc": {
+          "type": "string",
+          "maxLength": 1000
+        }
+      }
+    },
+    "Account": {
+      "type": "object",
+      "properties": {
+        "protocol": {
+          "type": "string",
+          "maxLength": 50
+        },
+        "host": {
+          "type": "string",
+          "maxLength": 150
+        },
+        "id": {
+          "type": "string",
+          "maxLength": 450
+        },
+        "claim": {
+          "type": "string",
+          "maxLength": 450
+        },
+        "attestations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Attestation"
+          }
+        }
+      },
+      "required": [
+        "protocol",
+        "host",
+        "id"
+      ]
+    }
+  }
+}
+```
+
+### Tags
+
+In the config of the *definition* defined above we can see a `tags` property. We simply apply the tags in the config to the `tags` property in the *metadata* of the *reference* Tile.
+
+### Example
+
+An example Social Accounts *reference* document that includes two accounts, Twitter and Github.
+
+```js
+const profile = await ceramic.createDocument('tile', {
+  metadata: {
+    schema: "<reference-schema-DocID>"
+    tags: ["AlsoKnownAs"]
+  },
+  content: [{
+    protocol: "https",
+    host: "https://twitter.com",
+    id: "https://twitter.com/marysmith",
+    // ID of tweet containing the user's DID
+    claim: "https://twitter.com/marysmith/status/1274020265417076736",
+    attestations: [{ did-jwt-vc: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..." }]
+  }, {
+    protocol: "https",
+    host: "https://github.com",
+    id: "https://github.com/marysmith",
+    // ID of Gist containing the user's DID
+    claim: "https://gist.github.com/marysmith/5c48debdb7089b3c8f86cca31739572c",
+    attestations: [{ did-jwt-vc: "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9..." }]
+  }]
+})
+```
+
+
+## Suggested Usage
+
+It is recomended to always include both the `claim` and the `attestation` property. The claim is usually a way to verify that the social account actually want's to be associated with the users DID. The attestation is a [Verifiable Credential](https://github.com/decentralized-identity/did-jwt-vc) from a third party that this is the case. In some scenarious it might be impossible to provide a publicly verifiable `claim`, for example it's not possible to create a public post which can be linked to in an easy. In these cases just an `attestation` will have to suffice. Note however that this means that there is full trust in the issuer of this attestation, where as when there is a public claim it's always possible to double check the truth of the attestation.
```

Comment:
> In some scenarious it might be impossible to provide a publicly verifiable `claim`, for example it's not possible to create a public post which can be linked to in an easy

in an easy...?

Perhaps something like "for example, in some systems it may not be possible to create a public post which can be easily linked to"

Reply:
ty